model_fitting/module_wraper.py

```diff
@@ -303,9 +304,10 @@ def build_classifier(reads_path, motifs_path, model_path, logs_dir, sample2bc, n
         f.write(' '.join(argv) + '\n')
 
 
-def get_faa_file_name_from_path(path, use_mapitope):
+def get_faa_file_name_from_path(path, use_mapitope, use_unique_faa_scanning):
     for file_name in os.listdir(path):
-        if file_name.endswith('faa') and 'unique' in file_name and ('mapitope' in file_name) == use_mapitope:
+        if file_name.endswith('faa') and (('unique' in file_name and use_unique_faa_scanning) or ('unique' not in file_name and not use_unique_faa_scanning)) \

```

Comment:
```('unique' in file_name) == use_unique_faa_scaning``` is shorter and clearer

Reply:
use_rpm_faa_scanning means to use the file unique_rpm.faa, but the `'unique' in file_name` find the right file between all the faa files in the folder.

Reply:
OK but ` (('unique' in file_name and use_unique_faa_scanning) or ('unique' not in file_name and not use_unique_faa_scanning))` still equals to `('unique' in file_name) == use_unique_faa_scaning`